<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_tienda</class>
 <widget class="QDialog" name="ventana_tienda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Acción</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>     |      </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Acción</string>
   </property>
  </widget>
  <widget class="QLabel" name="azada">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/otros/hoe.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hacha">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/otros/axe.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="semillas_choclo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>55</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/cultivos/choclo/seeds.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="semillas_alcachofa">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>55</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/cultivos/alcachofa/seeds.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$ 50</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$ 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$ 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_comprar_azada">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Comprar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_vender_azada">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_comprar_hacha">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Comprar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_vende_hacha">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_comprar_s_choclo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Comprar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_vender_s_choclo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_comprar_s_alcachofa">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Comprar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_vender_s_alcachofa">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>55</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/cultivos/alcachofa/icon.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>55</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/cultivos/choclo/icon.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>55</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/recursos/gold.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/recursos/wood.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_vender_alcachofa">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_vender_choclo">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_vender_madera">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_vender_oro">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$ 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$ 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$ 20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$ 40</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/otros/window_template.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>55</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/otros/ticket.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>$ 1500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_ticket">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Comprar</string>
   </property>
  </widget>
  <zorder>label_24</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>azada</zorder>
  <zorder>hacha</zorder>
  <zorder>semillas_choclo</zorder>
  <zorder>semillas_alcachofa</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>boton_comprar_azada</zorder>
  <zorder>boton_vender_azada</zorder>
  <zorder>boton_comprar_hacha</zorder>
  <zorder>boton_vende_hacha</zorder>
  <zorder>boton_comprar_s_choclo</zorder>
  <zorder>boton_vender_s_choclo</zorder>
  <zorder>boton_comprar_s_alcachofa</zorder>
  <zorder>boton_vender_s_alcachofa</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>boton_vender_alcachofa</zorder>
  <zorder>boton_vender_choclo</zorder>
  <zorder>boton_vender_madera</zorder>
  <zorder>boton_vender_oro</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>boton_ticket</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
